Documentation/Setup/Config/Index.rst

```diff
@@ -1259,7 +1264,57 @@ htmlTag\_setParams
 
             config.htmlTag_setParams = xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-US"
 
+.. warning::
+    If you are using `htmlTag.atttributes` this property (`htmlTag_setParams`) will not have any effect.
+
+.. note::
+    Used to older TYPO3 versions? If you are using site configuration you do not need to set `htmlTag_setParams` for language 
+    related configuration in TypoScript - use the site configuration instead.
+
+htmlTag.attributes
+==================
+
+.. container:: table-row
+
+   Property
+         htmlTag.attributes
+
+   Data type
+         :ref:`data-type-array`
+
+   Description
+         Sets the attributes for the :html:`<html>` tag on the page. Allows to override 
+         and add custom attributes via TypoScript without having to re-add the 
+         existing attributes generated by SiteHandling.
+
+         This property supersedes the previous `config.htmlTag_setParams` option by providing
```

Comment:
does it make sense to write this at htmlTag_setParams as well?

Reply:
htmlTag_setParams got these two infos:


.. warning::
    If you are using `htmlTag.atttributes` this property (`htmlTag_setParams`) will not have any effect.

.. note::
    Used to older TYPO3 versions? If you are using site configuration you do not need to set `htmlTag_setParams` for language 
    related configuration in TypoScript - use the site configuration instead.